controllers/repository_controller.go

```diff
@@ -291,3 +295,30 @@ func (r RepositoryReconciler) ensureRatingServiceAccount(ctx context.Context, na
 
 	return nil
 }
+func (r *RepositoryReconciler) getRepoType(repo *corev1alpha1.Repository) bool {
```

Comment:
why did you use a `bool` to represent a repo type???? You should return (string,error) ... 
The returns should match your function name. That's how you define/develop a function.